### DISCUSSION TITLE
Room for improving site search functionality?

### DISCUSSION THREAD

**Discussion by akhmerov** [executablebooks/meta#176] [Ideas]
Jupyter book now uses the built-in sphinx site search. It's OK, but it hasn't been improved in a long while. For an illustration of what search could look like check out https://squidfunk.github.io/mkdocs-material/

Do you think there's some way to improve on what sphinx currently has to offer?

**Comment by choldgraf** [comment]
I suspect that it's possible to improve this - in a worst case scenario one could always totally over-ride Sphinx's search functionality with something like `lunr.js`.
